### DISCUSSION TITLE
[x][closed] track:content_type

### DISCUSSION THREAD

**Discussion by dzerbino** [fairtracks/fairtracks_standard#94] [Team Posts]
track:content_type is required, but nothing in content_type is required, so it could well be empty.

**Comment by sveinugu** [comment]
Ah. Schema bug. Noted

**Comment by sveinugu** [comment]
@dzerbino So even though I mistakenly forgot to make this a required attribute, did not mean that you could take the easy way out and skip it! ;D

Seriously, though, I see that it is quite hard to fill out as the definition is quite loose. However, I think it would be useful for an upcoming EDAM discussion if you gave it a try?

**Comment by dzerbino** [comment]
Obviously I didn't want to update the trackhub before you updated the spec ;P

However, the definition is indeed rather loose, and I'm rather at a loss as to what to put in there. For example if I were to lookup 'sequence coverage' the best term appears in the influenza ontology of all places:
https://www.ebi.ac.uk/ols/ontologies/flu/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FFLU_0000848

**Comment by sveinugu** [comment]
Created an issue to fix this: https://github.com/fairtracks/fairtracks_standard/issues/5

**Comment by sveinugu** [comment]
Added "Make attribute required" as action point on issue, with back reference. Removing pin
